R/sql_methods.R

```diff
@@ -13,7 +14,7 @@
 #' db <- read_sql("SELECT * FROM table")
 #' db <- read_sql("SELECT * FROM table", dbname = "space_1", schemaname = "test_schema")
 #' @export
-read_sql <- function(sql, dbname = NULL, schemaname = NULL){
+read_sql <- function(sql, index_col = NULL, dbname = NULL, schemaname = NULL){
```

Comment:
Please always add new parameters as the final argument, to avoid breaking existing position calls

Reply:
corrected.